package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jshint romanizer.test.js"
+    "lint": "jshint romanizer.js",
+    "test": "npm run lint && tape romanizer.test.js | tap-spec"
```

Comment:
Thanks for editing :)
that's what we were searching for the whole time "how to run more than one module with same command at the same time"

Reply:
I might leave this open and demonstrate it to people during the projects